### PR TITLE
ref(strophe): Remove and cleanup all strophe related code.

### PR DESCRIPTION
**Pull request by hristoterezov**
Everything strophe related is now handled in lib-jitsi-meet project.

Depends on https://github.com/jitsi/lib-jitsi-meet/pull/614

### CODE REVIEW

File: react/features/base/participants/functions.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
 // @flow
+import md5 from 'js-md5';
```
Comment (saghul) [inline react/features/base/participants/functions.js]: where was MD5 coming from before?

File: react/features/base/lib-jitsi-meet/native/polyfills-browser.js [hunk 2]
```diff
@@ -143,24 +143,10 @@ function _visitNode(node, callback) {
             document.cookie = '';
         }
 
-        // Document.querySelector
```
Comment (saghul) [inline react/features/base/lib-jitsi-meet/native/polyfills-browser.js]: Why is this not needed anymore? Do we no longer use the caps plugin?

Comment (hristoterezov) [reply react/features/base/lib-jitsi-meet/native/polyfills-browser.js]: Yes. We don't use it!

File: react/features/base/participants/functions.js (hunk 1 above)
Comment (hristoterezov) [reply react/features/base/participants/functions.js]: Strophe.

**Comment by hristoterezov** [comment]
@saghul I've just rebased it! I've tested mobile and it appears to work!

**Pull request merged**